modules/qna/src/views/full/FormModal.jsx

```diff
@@ -56,7 +58,13 @@ export default class FormModal extends Component {
     })
   }
 
-  closeAndClear = () => {
+  closeAndClear = (modified) => {
+    //Will delete created contentElements if the user isn't going to save the changes
+    if (modified != true) {
+      if (this.state.createdContentElements.length > 0) {
```

Comment:
you can replace this with only ` if (this.state.createdContentElements.length) { `